topics/statistics/tutorials/galaxy-ludwig/tutorial.md

```diff
@@ -0,0 +1,467 @@
+---
+layout: tutorial_hands_on
+level: Intermediate
+title: Train and Test a Deep learning image classifier with Galaxy-Ludwig
+questions:
+- How do I use the Ludwig tool to create and run a deep learning image classifier? 
+- What files do I need to run Ludwig? 
+- How do I build and evaluate the image classification model? 
+objectives:
+- Execute a Ludwig experiment. 
+- Evaluate the image classfier model. 
+- Compare different configurations to improve model performance. 
+time_estimation: 40M
+key_points:
+- Use Galaxy-Ludwig Experiment to solve an image classification problem
+- Using CNN on MNIST dataset. 
+contributors:
+- paulocilasjr 
+- qchiujunhao 
+- jgoecks 
+tags:
+- MNIST
+- Deep learning
+- Ludwig
+---
+
+> <comment-title>Galaxy-Ludwig Tool</comment-title>
+>
+> The Ludwig tool described in this tutorial is only available at: 
+> ```https://cancer.usegalaxy.org/```
+{:  .comment}
+
+Using the MNIST image dataset of handwritten digits as input, we will build an image recognition model with the Galaxy-Ludwig tool. The goal is to classify the handwritten digit in each image.
+
+To accomplish this, three steps are needed: (i) upload Ludwig files and image files to Galaxy (ii) Set up and running the Ludwig experiment function on Galaxy, and (iii) Evaluate the image classification model. As a bonus step, we'll also explore (iv) improving the model's classification performance (Figure 1).
+
+![schema of the whole process of training model and test.](../../images/galaxy-ludwig/explain_model_schema.png "Overview of the steps process to obtain the handwritten classification model and testing it.")
+<!-- You may want to cite some publications; this can be done by adding citations to the
+bibliography file (`tutorial.bib` file next to your `tutorial.md` file). These citations
+must be in bibtex format. If you have the DOI for the paper you wish to cite, you can
+get the corresponding bibtex entry using [doi2bib.org](https://doi2bib.org).
+
+With the example you will find in the `tutorial.bib` file, you can add a citation to
+this article here in your tutorial like this:
+{% raw %} `{% cite Batut2018 %}`{% endraw %}.
+This will be rendered like this: {% cite Batut2018 %}, and links to a
+[bibliography section](#bibliography) which will automatically be created at the end of the
+tutorial.-->
+
+> <agenda-title></agenda-title>
+>
+> In this tutorial, we will cover:
+>
+> 1. TOC
+> {:toc}
+>
+{: .agenda}
+
+
+> <comment-title>Background</comment-title>
+>
+> [MNIST dataset](https://yann.lecun.com/exdb/mnist/) provides the database of handwritten digits
+> the database has a training set of 60,000 examples, and a test set of 10,000 examples.
+> at our planet and its environment for the ultimate benefit of all European citizens.
+> It is a subset of a larger set available from NIST.
+> The digits have been size normalized and centered in a fixed-size image.
+{:  .comment}
+
+# FILES FORMAT
```

Comment:
please don't use all caps for the section headings

Reply:
No problem! I will change them all.